massgov/mayflower-patternlab
# DP-8378 The page banner display incorrectly for topic page within mobile responsive

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description


## Related Issue / Ticket

- [DP-8378 The page banner display incorrectly for topic page within mobile responsive](https://jira.state.ma.us/browse/DP-8378)

## Steps to Test


1. Open pages contain `div.ma__page-banner__container`,  Topic, Service and Org pages.
2. Check the header banner display in different window sizes.
- Banner background shapes
- Banner overlay covers both sides in small displays. 
- In Topic page, if an icon is available, make sure it's displayed centered in the title height.
- Not only the page title, but make sure the banner description is fully displayed.

## Screenshots
Off centered icon:
![offcentered_icon](https://user-images.githubusercontent.com/9633303/38034686-d15e1484-3270-11e8-9723-1d372a0b86e8.png)
Missing description:
![org_missing_desc2](https://user-images.githubusercontent.com/9633303/38034687-d1763f64-3270-11e8-8b0b-456457abe4a0.png)
Missing description:
![org_missing_desc](https://user-images.githubusercontent.com/9633303/38034688-d182d29c-3270-11e8-98c7-754b0cd873b0.png)
Overlay layer is on the top of the title and the description:
![service1](https://user-images.githubusercontent.com/9633303/38034689-d190050c-3270-11e8-9491-e3fbff847cfe.png)
Description cuf-off:
![topic_desc-cutoff](https://user-images.githubusercontent.com/9633303/38034690-d19ef5d0-3270-11e8-9cf9-a6a8d05ebb91.png)
Overlay positioning:
![org_overlay_positioning](https://user-images.githubusercontent.com/9633303/38034691-d1b0a2bc-3270-11e8-8b53-a22d7d740986.png)


## Additional Notes:

In Drupal side, [DP-8378 The page banner display incorrectly for topic page within mobile responsive #2060](https://github.com/massgov/mass/pull/2060) needs to be released at the release of this change to remove override CSS as hotfix in Drupal.

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
